agent_shared/src/config.rs

```diff
@@ -14,6 +16,15 @@ use strum::VariantArray;
 use tracing::{debug, info};
 use url::Url;
 
+// TODO: the stronghold path does not need to be configured through the config file anymore. Is this static variable for
+// the stronghold path the right solution?
+static STRONGHOLD_PATH: &str = "./app/res/stronghold";
```

Comment:
The path to the Stronghold file should still be an optional config parameter to allow users to choose the location, if they want to. Also, please add a file extension (such as `.dat`) to the default. Also, /res/ seems a bit arbitrary to me as it reflects the source folder structure, but in the final container "./app/stronghold.dat" should be enough.

Reply:
The path to the stronghold file is still an optional config (removed the TODO comment). 

I changed the default value to your suggested value :+1: